cell.rb

```diff
@@ -1,22 +1,17 @@
 class Cell
   attr_accessor :status
   def initialize
-    @status = 'Dead'
+    @status = (rand(2)) == 1 ? true : false
   end
-
-  def is_alive
-    if @status == 'Alive'
-      return true
-    else
-      return false
-    end
+  
+  def get_print_char
+    return 'O' if alive?
+    'X'
   end
 
-  def get_print_char
-    if is_alive()
-      return 'O'
-    else
-      return 'X'
-    end
+  private
+
+  def alive?
+    return @status
```

Comment:
It is not necessary to specify the return in this case

Reply:
Fixed, thanks!